etc/gummi.profile

```diff
@@ -0,0 +1,17 @@
+# Firejail profile for gummi
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gummi.local
+# Persistent global definitions
+# added by caller profile
+#include globals.local
+
```

Comment:
Has gummi any config dirs like `~/.config/gummi` or `~/.local/share/gummi`?

Reply:
```~/.config/gummi``` yes

Reply:
- add a `noblacklist ${HOME}/.config/gummi` here and a `blacklist ${HOME}/.config/gummi` in disable-programs.inc.
- to see if there are more run `firejail --name=gummi --ignore=private-bin --profile=path/to/gummi.profile --private gummi` in one termial, then do some things in gummi like open and save a template, changes settings, .... then run `firejail --join=gummi tree -a` (or `ls -aR` if you haven't installed tree).

Reply:
As I'm seeing it now and it is inconsistent in all profiles in etc:  
Shouldn't ```#include globals.local``` be ```include globals.local```  for all programs?

Reply:
@florianbegusch We need to avoid including 'globals.local' more than once in all profiles. So when a profile redirects to another profile that already includes global.local, we have to comment the include in the first one. This is standard firejail procedure.

Reply:
> @florianbegusch We need to avoid including 'globals.local' more than once in all profiles. So when a profile redirects to another profile that already includes global.local, we have to comment the include in the first one. This is standard firejail procedure. So yes, only latex-common.profile needs
> 
> ```
> # added by caller profile
> #include globals.local
> ```
> 
> All the others are the caller profiles, so they can't have that or globals.local never gets called.

Thank you for the quick response! Really appreciate it.